# Oneirc comes with some DSO linking changes, including --as-needed
# which broke the HTML <video> tag. The way libffmpegsumo.so was linked,
# libvpx ended up dropped from the shared lib.
# See http://wiki.debian.org/ToolChain/DSOLinking
---
 third_party/ffmpeg/ffmpeg.gyp |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

Index: src/third_party/ffmpeg/ffmpeg.gyp
===================================================================
--- src.orig/third_party/ffmpeg/ffmpeg.gyp
+++ src/third_party/ffmpeg/ffmpeg.gyp
@@ -357,18 +357,18 @@
                 ],
                 'libraries': [
                   '-lz',
                 ],
                 'conditions': [
                   ['use_system_vpx==1 and target_arch!="arm"',
                    {
                      # Using libvpx provided by the system.
-                     'ldflags': [
-                       '-L/usr/local/lib', '-lvpx',
+                     'libraries': [
+                       '-lvpx',
                      ],
                    }],
                 ],
               },
               'variables': {
                 'obj_format': 'elf',
                 'conditions': [
                   [ 'use_system_yasm==1', {
